src/Symfony/Component/HttpKernel/Tests/EventListener/RouterListenerTest.php

```diff
@@ -299,7 +299,10 @@ public static function provideRouteMapping(): iterable
             [
                 'slug' => 'vienna-2024',
                 '_route_mapping' => [
-                    'slug' => 'conference',
```

Comment:
this should still be working, right? If so, please add another test case and keep this one